src/tsserver/nodeServer.ts

```diff
@@ -705,8 +705,8 @@ namespace ts.server {
                     cancellationToken,
                     ...options,
                     typingsInstaller: typingsInstaller || nullTypingsInstaller,
-                    byteLength: Buffer.byteLength,
-                    hrtime: process.hrtime,
+                    byteLength: Buffer.byteLength as (buf: string, encoding?: string | undefined) => number,
+                    hrtime: process.hrtime as (start?: number[] | undefined) => number[],

```

Comment:
These types are APIs where the node version is not assignable to ours when function strictness is enabled. I don't see a great solution here.

Reply:
I changed `hrtime` because that seems like the correct thing to do. I'd change `byteLength` to use `ts.BufferEncoding` as its parameter, but none of our other interfaces use it for things like file reading (where I would think it'd be a good idea).